recipes/libcugraph/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "libcugraph" %}
+{% set version = "0.19.0a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/rapidsai/cugraph/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: eea8eb788ee2e9f369b6d4911c8cb7c4819453e17aa855eac14af86ee27f627e
+    folder: cugraph
+  # libcugraph depends on the RMM header-only library of the same version.
+  - url: https://github.com/rapidsai/rmm/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: aa2b6542e51e9d3bec1c4053ab2a06af8fb8e9a94be3832418109904ebebb276
+    folder: rmm
+
+build:
+  number: 0
+  skip: true  # [(not linux) or (cuda_compiler_version == "None")]
+  run_exports:
+    - {{ pin_subpackage(name, max_pin="x.x") }}
+  missing_dso_whitelist:
+    - "*/libcuda.*"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}
+    - cmake >=3.12.4
+    - make
+    - libgomp
+  host:
+    - boost-cpp
+    - nccl
+    - libfaiss =*=*_cuda
+    - spdlog =1.7.0
+
+test:
+  commands:
+    # Only testing for select headers and the library
+    - test -f $PREFIX/include/cugraph/version_config.hpp
+    - test -f $PREFIX/include/cugraph/algorithms.hpp
+    - test -f $PREFIX/include/cugraph/graph.hpp
+    - test -f $PREFIX/lib/libcugraph.so
+
+about:
+  home: https://github.com/rapidsai/cugraph
+  license: Apache-2.0
+  license_file:
+    - cugraph/LICENSE
+    - rmm/LICENSE
```

Comment:
Also need the license files of gunrock, thrust, cuco, etc. You can see all the projects in https://github.com/rapidsai/cugraph/blob/f5ba9e7a33fea5a69949283a5de33a4eb1233c36/cpp/CMakeLists.txt

Reply:
Thanks, I've specified them in the latest commit. The CI log seems to indicate that license files were included, but if I need to include those licenses differently (eg. have the build script copy them to a single license directory for example), then please let me know.  @isuruf , also all licenses are Apache 2.0 except for cuHornet which is BSD .  Do I need to change the `license:` entry to mention that (eg. a comma-separated string) or is it acceptable to have it mention only the license of `libcugraph` itself (similar question also brought up [here](https://github.com/conda/conda-build/pull/3616#issuecomment-513036791))?

Reply:
`Apache-2.0 AND BSD-3-Clause` (or BSD-2-Clause depending on the license)

Reply:
Thanks, yes it's 3-clause. I just updated it in the latest commit.